base/range.jl

```diff
@@ -115,9 +124,54 @@ convert(::Type{T}, r::AbstractRange) where {T<:AbstractRange} = r isa T ? r : T(
 
 ## ordinal ranges
 
+"""
+    OrdinalRange{T, S} <: AbstractRange{T}
+
+Supertype for ordinal ranges with elements of type `T` with
+spacing(s) of type `S`.
+The spacing of the elements of the range does not need to be even. The steps
```

Comment:
I'm not sure what "The spacing of the elements of the range does not need to be even" means.  Maybe just remove this sentence?